### PR TITLE
fix(moe): pad NVFP4 expert shards for TRT-LLM

### PR DESCRIPTION
**Pull request by tuanzhangCS**
## Summary

Under MoE TP, an NVFP4 expert's per-rank intermediate size may not be a multiple of 64 (e.g. a 640-wide expert under TP4 gives 160 per rank), which the `flashinfer_trtllm` NVFP4 weight layout does not accept. This PR pads NVFP4 expert shards so the kernel's ISPP alignment is always met:

- Declare `ispp_alignment = 64` (`TRTLLM_NVFP4_ISPP_ALIGNMENT`) for the three flashinfer TRT-LLM NVFP4 kernels (apply / routed apply / SiTU routed apply) in the kernel registry.
- Apply the existing TRT-LLM ISPP padding path for `_quant_kind == "nvfp4"` in the MoE expert layer, so padding happens on the global intermediate size before TP sharding.
- Allocate NVFP4 packed weights and block scales with `torch.zeros` instead of `torch.empty`, so the padded tail is zero-filled and does not change the MoE result.
- Document the behavior in `docs/serving/parallelism.md`.

Tests added:
- Runtime test that the packed NVFP4 weight and block-scale loaders share the same padded TP shard and the padded tail stays zero.
- Kernel test parametrized with `logical_ispp=160` (padded to 192) verifying the SiTU NVFP4 MoE still matches the dequant reference on a padded shard.
- Registry test asserting all three NVFP4 TRT-LLM kernels declare `ispp_alignment == 64`.

## Test Plan

- `pytest test/runtime/test_moe_global_padded_tp_loader.py`
- `pytest tokenspeed-kernel/test/ops/moe/test_nvfp4_situ_flashinfer_trtllm.py`
- E2E: qwen3.8-flash-next NVFP4 with MoE TP4, gsm8k eval

### CODE REVIEW

File: python/tokenspeed/runtime/layers/moe/weights/nvfp4.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def create_nvfp4_weight_pair(
 ) -> None:
     ispp = spec.intermediate_size // spec.tp_size
     w13_weight = torch.nn.Parameter(
-        torch.empty(
+        torch.zeros(
```
[the remark this answers is not in the document]
Comment (tuanzhangCS) [reply python/tokenspeed/runtime/layers/moe/weights/nvfp4.py:43]: Zeros are only needed for the padded tail. The loader overwrites the valid checkpoint region, while the padded rows/columns remain unwritten; initializing them to zero ensures they do not affect the MoE output.
